platform/android/java/build.gradle

```diff
@@ -233,8 +233,7 @@ def generateBuildTasks(String flavor = "template") {
 }
 
 def isAndroidStudio() {
-    def sysProps = System.getProperties()
-    return sysProps != null && sysProps['idea.platform.prefix'] != null
+    return project.hasProperty('android.injected.invoked.from.ide')
```

Comment:
Can you apply the fix in [config.gradle](https://github.com/godotengine/godot/blob/40b4130c93d08235a60996d29e5869a22b6ae53d/platform/android/java/app/config.gradle#L332) as well.

Is this approach supported by past releases of Android Studio?

Reply:
Actually looking at the code, the definition of `isAndroidStudio()` defined in `config.gradle` is visible here as well, so you can just make the change in the version in `config.gradle` and delete the version of `isAndroidStudio()` that's in this `build.gradle` file.

Reply:
> can just make the change in the version in config.gradle and delete the version of isAndroidStudio() that's in this build.gradle file.

Done. 

> Is this approach supported by past releases of Android Studio?

This property has been available since 2014: https://android.googlesource.com/platform/tools/base/+/45573717b3f70c58b9e389ada306b25d477af6d9